modules/analytics/analytics.js

```diff
@@ -123,6 +123,13 @@ class Analytics {
                     permanentProperties.group = group;
                 }
 
+                if (window.jitsiRegionInfo) {
```

Comment:
The agreement is about not having global variables in new code, pretty much. Now, I understand this is not new code, so we can let this one slide and refactor things together with the Electron desktop picker, for example.